### DISCUSSION TITLE
Store data in MongoDb

### DISCUSSION THREAD

**Discussion by tomerten** [joqueka/bf4py#11] [Ideas]
By adding a small function in combination with the generator (see discussion #10) and a json flattener it is easy to store the key data, details and datasheet headers for equity in a MongoDb for later reference.  FYI: This allows me to build a list of Yahoo Finance Ticker symbols  and MorningStar Ticker symbols from ISIN numbers.

Would you be interested in adding this functionality?

**Comment by joqueka** [comment]
I personally prefer using pandas and HDF files, that's the reason why I return only raw dict values. So everyone can convert it to a format he or she prefers. However I like the idea of building some kind of adapter that converts the data to another format, I'll think about that.

Converting an ISIN to a ticker symbol should not be more than two lines of code right? Right now I see the main task of my package to provide data that is available on boerse-frankfurt without taking into account what you can do with this data.

But feel free to post any code that you think could be valuable for others.
